daisy_workflows/image_test/network/network-testee.bat

```diff
@@ -0,0 +1,40 @@
+@echo off
```

Comment:
Just make this startup script powershell to begin with.

Reply:
lol, you're right